osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -486,8 +503,46 @@ public bool OnPressed(GlobalAction action)
 
         public void OnReleased(GlobalAction action)
         {
+            switch (action)
+            {
+                case GlobalAction.SelectNext:
+                case GlobalAction.SelectPrevious:
+                    endRepeatSelection(action);
+                    break;
+            }
         }
 
+        private const double repeat_interval = 120;
+
+        private ScheduledDelegate repeatDelegate;
+        private object lastRepeatSource;
+
+        /// <summary>
+        /// Begin repeating the specified selection action.
+        /// </summary>
+        /// <param name="action">The action to perform.</param>
+        /// <param name="source">The source of the action. Used in conjunction with <see cref="endRepeatSelection"/> to only cancel the correct action (most recently pressed key).</param>
+        private void beginRepeatSelection(Action action, object source)
+        {
+            endRepeatSelection();
+
+            lastRepeatSource = source;
+            Scheduler.Add(repeatDelegate = new ScheduledDelegate(action, Time.Current, repeat_interval));
```

Comment:
See: `DrawableExtensions.BeginKeyRepeat()`.

Reply:
wow, that exists. will have to check how the delays feel.